### PR TITLE
enh: better stats + désactivate comment

### PR DESCRIPTION
**Pull request by armand-dfl**
# Pull Request

### Description

I fix the certification request stat + you can now desactivate and activate a comment + you can now see the number of comment in attempt of validation

### Pull Request Type

-   [x] Feature
-   [ ] Fix
-   [ ] Hotfix
-   [ ] Docker

### By pushing this PR i assumed that i've:

-   [x] Tested My Code
-   [x] Did it a way it is reusable
-   [x] Documented the coode
-   [x] Moved the Trello Card

### CODE REVIEW

File: www/Controller/Admin.class.php [hunk 1]
```diff
@@ -47,11 +49,15 @@ public function dashboard()
...
 
-        $totalCertificationDemand = $certification->select([
+        /*$totalCertificationDemand = $certification->select([
             "certification" => [
                 "args" => ["COUNT(*)"],
             ]
-        ]);
+        ]);*/
+
+        $totalCertificationDemand = $certification->select2('certification', ['COUNT(*) as total'])
```
Comment (NoePigeau) [inline www/Controller/Admin.class.php]: antoine a mis en place une method count(), tu as juste a remplacer fetch par count

File: www/Controller/Admin.class.php [hunk 2]
```diff
@@ -62,11 +68,18 @@ public function dashboard()
             ]
         ]);
 
+        $totalCommentDemand = $comment->select2('comment', ['COUNT(*) as total'])
```
Comment (NoePigeau) [inline www/Controller/Admin.class.php]: pareil ici

File: www/Controller/Admin.class.php (hunk 1 above)
Comment (armand-dfl) [reply www/Controller/Admin.class.php]: Modifié

File: www/Controller/Admin.class.php (hunk 2 above)
Comment (armand-dfl) [reply www/Controller/Admin.class.php]: Modifié

**Review by NoePigeau — APPROVED** [review]

**Pull request merged**